lib/super_digit.rb

```diff
@@ -1,9 +1,12 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n), where n is number of digits
+# Space Complexity - O(n), where n is number of digits, because of call stack
 def super_digit(n)
-   
+    # base case
+    return n if n < 10
+    # recursive case
+    return n%10 + super_digit(n/10) 
```

Comment:
This doesn't quite work, 
consider:

```suggestion
    # base case
    return n if n < 10
    # recursive case
    answer =  n%10 + super_digit(n/10) 
    while(answer >= 10)
      answer =  answer%10 + super_digit(answer/10)
    end
    return answer 
```